homeassistant/components/teslemetry/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "teslemetry",
+  "name": "Teslemetry",
+  "codeowners": ["@Bre77"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/teslemetry",
+  "iot_class": "cloud_polling",
+  "loggers": ["teslemetry"],
```

Comment:
Is this the correct logger?

Reply:
I have no idea, ive only ever seen this set to the integration domain. The library doesnt use a logger.

Reply:
```suggestion
  "loggers": ["tesla-fleet-api"],
```